app/helpers/application_helper.rb

```diff
@@ -13,4 +13,14 @@ def toastr_flash
     end
     safe_join @message
   end
+
+  def average_rating ratings
+    if ratings.present?
+      (ratings.map{|x| x.point.to_i}.sum.to_f / ratings.size).round(
+        Settings.rouding
+      )
```

Comment:
point kiểu dữ liệu là integer rồi thì ko cần to_i nữa e
```suggestion
      (ratings.sum(&:point).to_f / ratings.size).round Settings.rouding
```

Reply:
đổi sang float để khi chia kết quả ra float ý anh

Reply:
`
[1] pry(main)> 2/3
=> 0
[2] pry(main)> 2.to_f/3
=> 0.6666666666666666
`

Reply:
`[1] pry(main)> 2/3
=> 0
[2] pry(main)> 2.to_f/3
=> 0.6666666666666666`